pkg/rvtools/parser.go

```diff
@@ -0,0 +1,147 @@
+package rvtools
+
+import (
+	"bytes"
+	"fmt"
+	"slices"
+
+	api "github.com/kubev2v/migration-planner/api/v1alpha1"
+	"github.com/xuri/excelize/v2"
+)
+
+func ParseRVTools(rvtoolsContent []byte) (*api.Inventory, error) {
+	excelFile, err := excelize.OpenReader(bytes.NewReader(rvtoolsContent))
+	if err != nil {
+		return nil, fmt.Errorf("error opening Excel file: %v", err)
+	}
+	defer excelFile.Close()
+	
+	sheets := excelFile.GetSheetList()
+
+	inventory := api.Inventory{}
+
+	inventory.Infra.Datastores = []api.Datastore{}
+	inventory.Infra.HostsPerCluster = []int{}
+	inventory.Vms.MigrationWarnings = api.MigrationIssues{}
+	inventory.Vms.NotMigratableReasons = []struct {
+		Assessment string `json:"assessment"`
+		Count      int    `json:"count"`
+		Label      string `json:"label"`
+	}{}
+	
+	if slices.Contains(sheets, "vInfo") {
+		vcenterUUID, err := extractVCenterUUID(excelFile, "vInfo")
+		if err != nil {
```

Comment:
As you return in `extractVCenterUUID` empty string in case of error those `if` `else` are redudant